modules/tensor_mechanics/test/tests/shell/static/inclined_straintest.i

```diff
@@ -0,0 +1,378 @@
+# Static test for the inclined shell element.
+# A single shell element is oriented at a 45 deg. angle with respect to the Y axis.
+# One end of the shell is fixed and an axial deformation to the shell element is
+# applied at the other end by resolving the deformation into Y and Z direction.
+# The stress and strain result in the global orientation when transformed to
+# the shell oriention gives the correct value of the axial stress and strain.
+
+[Mesh]
+  type = FileMesh
+  file = shell_inclined.e
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Variables]
+  [./disp_x]
```

Comment:
Can you please use the current input file syntax for new files. (i.e. no leading `./` and `[../]` -> `[]`)

Reply:
Removed ./ in all the test  input files.